TheaterSchedule/android/app/src/main/java/com/awesomeproject/MainActivity.java

```diff
@@ -0,0 +1,15 @@
+package com.theaterschedule;
```

Comment:
Correct path or recreate project with correct name ( i think recreation is easier )